src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Application {
+    public static final int FIGURE_COUNT = 6;
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < FIGURE_COUNT; i++) {
+            if (i < FIGURE_COUNT / 2) {
+                figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+            } else {
+                figures[i] = FIGURE_SUPPLIER.getDefaultFigure();
+            }
+        }
+        for (Figure figure: figures) {
+            System.out.println(figure);
+        }
```

Comment:
1. according to task description we should use draw() method for printing info about figure
2. there is no need for second loop, we can do required within single one
```suggestion
        for (int i = 0; i < FIGURE_COUNT; i++) {
            if (i < FIGURE_COUNT / 2) {
                figures[i] = FIGURE_SUPPLIER.getRandomFigure();
            } else {
                figures[i] = FIGURE_SUPPLIER.getDefaultFigure();
            }
            figures[i].draw();
        }
```